### DISCUSSION TITLE
少年不识成功味 | /cn/2019/01/what-is-success/

### DISCUSSION THREAD

**Discussion by yihui** [rbind/yihui#1286] [Comments]
https://yihui.org/cn/2019/01/what-is-success/

[a comment by an automated account is not shown]

**Comment by ZaiyanHere** [comment]
看到the affection of children的时候，会心一笑。
会心一笑的感觉真好，谢谢益辉，谢谢爱默生。

**Reply by yihui** [reply]
单从这方面来说，我可能已经是世界上最“成功”的人，因为大部分见过我的小朋友都非常喜欢我（对此我也不完全明白是为什么）。几家朋友聚会时，经常是其他大人们在餐桌上吃饭聊天，而所有小朋友们跟着我在别的房间玩，而且一不小心我身上就挂满了小猴子，甩都甩不掉。你可能没见过一个人一口气背三个甚至四个小朋友的壮观景象（背一个、左右各挂一个、前面再挂一个在脖子上）。后来他们都戏称我为幼儿园园长。

节日快乐！
